src/object/obj_rpc.h

```diff
@@ -147,7 +147,8 @@ enum obj_rpc_flags {
 	((uint32_t)		(orw_map_version)	CRT_VAR) \
 	((uint64_t)		(orw_dkey_conflict)	CRT_VAR) \
 	((struct dtx_id)	(orw_dti_conflict)	CRT_VAR) \
-	((daos_size_t)		(orw_sizes)		CRT_ARRAY) \
+	((daos_size_t)		(orw_iod_sizes)		CRT_ARRAY) \
+	((daos_size_t)		(orw_data_sizes)	CRT_ARRAY) \
```

Comment:
I don't quite see why we want to introduce 'orw_nrs' and 'orw_data_sizes' to differentiate with inline fetch? (which uses 'orw_sgls') shouldn't we just use 'orw_sgls' for both inline and bulk fetch? So that client can share same code.

Reply:
by using sgl it will pack all data including the iov_buf, see crt_proc_d_sg_list_t().
so for bulk the sgl is NULL to avoid packing the long data.

Reply:
I see, thanks for the explanation.